Smith_Ben/Algorithms/pg50(removeblanks, acronyms).html

```diff
@@ -0,0 +1,41 @@
+<html>
+<head>
+  <title></title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script type="text/javascript">
+  /*
+  $(document).ready(function(){
+    // ... your assignment here
+  });
+  */
+
+  function(remove blanks){
+    var new = ' '
+    For(var i = 0; i < str.length; i++){
+      if(str[i] == ' '){
+      }
+      else{
+        new += str[i];
+      }
+    }
+    return new;
+  }
+
+  function(acronyms){
+    var split_arr = arr.split(' ');
```

Comment:
what is arr?  It isn't defined anywhere prior to this line.  This will throw an error.